### ISSUE TITLE
Setup

### ISSUE THREAD

**Issue by hjone72** [Supalio/php-id3tag-modifier#2]
Hi,

Just wondering if you'd mind giving me a few pointers to get this going?

Thanks,

**Comment by Supalio** [comment]
Hi there,
Just a heads up, this is a project I made for fun while learning Laravel so it isn't very good. I did not think it would get any attention at all.

If you want to test it, you can install it on your computer and access the index page.
The only page working for the moment is the "Rename files" page which allow you to recursively scan a directory for all the .mp3 files in it.

You can then rename them one by one using the interface and move them to another directory or to a trash directory.
The paths for the three directories are defined by static variables inside the RenameController.php, I will change them to a config file in the future.

I'm currently working on auto-suggesting prettier names for the files so you don't have to enter them manually for every file.

The part to edit the tags is not yet available, I did it in an old project but haven't had the time to migrate it in this one. I'll try to do it soon.

Feel free to make any comments or changes to the code

**Comment by hjone72** [comment]
Looks pretty cool so far! I got it all running. The main thing I was missing was the `composer install`. Haven't done to much with composer so I over looked it...

I also had to edit the Encryption Key in `config/app.js`. It was set to env('APP_KEY') but this only threw an error for me. I set it to a manual string and all was good.

The main thing I wanted was the ID3 tagging so I'd love to see you migrate it to your new project!

Thanks for the quick reply :)
